src/auth/googleauth.ts

```diff
@@ -83,13 +107,24 @@ export class GoogleAuth {
 
   cachedCredential: OAuth2Client|null = null;
 
+  private keyfileName?: string;
+  private scopes?: string|string[];
+
   /**
    * Export DefaultTransporter as a static property of the class.
    */
   static DefaultTransporter = DefaultTransporter;
 
+  constructor(opts?: GoogleAuthOptions) {
+    opts = opts || {};
+    this._cachedProjectId = opts.projectId || null;
+    this.keyfileName = opts.keyFilename;
```

Comment:
cringe cringe cringe. `keyFilename` is the correct capitalization as per the style guide. Is there a way to fix the field?

Reply:
fixed!